starter-core/src/main/java/io/micronaut/starter/feature/function/gcp/template/gcpFunctionReadme.rocker.raw

```diff
@@ -5,19 +5,21 @@
 Project project,
 Features features,
 String runCommand,
-String buildCommand
+String buildCommand,
+boolean isFunction
 )
 
 # Micronaut and Google Cloud Function
 
+@if(!isFunction){
 ## Running the function locally
 
 ```cmd
 ./@runCommand
 ```
 
-And visit http://localhost:8080/hello
```

Comment:
Instead of deleting `/hello` we should probably replace it with `/@project.getPropertyName()`

Reply:
I've added `/@project.getPropertyName()`